scripts/beefy_signature_sampling.py

```diff
@@ -15,8 +15,7 @@ def samples(ratio_per_validator, validators_length, slash_rate, randao_commit_ex
 
     result += ceil(log2(validators_length));
 
-    if(signature_use_count > 0):
-        result += 1 + 2 * ceil(log2(signature_use_count))
+    result += 1 + (2 * ceil(log2(signature_use_count)) if signature_use_count > 1 else 0)
```

Comment:
This reflects the new math. The main difference is that there is one extra signature required when signature count is 0. This is because both the python and solidity code only added 1 if the signature count was greater than 0.

Old math:

Signature Use Count|Total|Static|Dynamic
-|-|-|-
0|27|18|9
1|28|18|10
2|30|18|12
3|32|18|14
4|32|18|14
5|34|18|16
6|34|18|16
7|34|18|16
8|34|18|16
9|36|18|18
10|36|18|18
11|36|18|18
12|36|18|18
13|36|18|18
14|36|18|18
15|36|18|18
16|36|18|18
17|38|18|20
18|38|18|20
19|38|18|20

Signature Use Count|Total|Static|Dynamic
-|-|-|-
0|28|18|10
1|28|18|10
2|30|18|12
3|32|18|14
4|32|18|14
5|34|18|16
6|34|18|16
7|34|18|16
8|34|18|16
9|36|18|18
10|36|18|18
11|36|18|18
12|36|18|18
13|36|18|18
14|36|18|18
15|36|18|18
16|36|18|18
17|38|18|20
18|38|18|20
19|38|18|20